src/main/java/net/mcreator/ui/ide/RSyntaxTextAreaStyler.java

```diff
@@ -18,30 +18,49 @@
 
 package net.mcreator.ui.ide;
 
+import net.mcreator.plugin.PluginLoader;
 import net.mcreator.preferences.PreferencesManager;
-import net.mcreator.ui.laf.AbstractMCreatorTheme;
+import net.mcreator.themes.ThemeLoader;
+import net.mcreator.ui.laf.MCreatorTheme;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.fife.ui.rsyntaxtextarea.RSyntaxTextArea;
 import org.fife.ui.rsyntaxtextarea.Style;
 import org.fife.ui.rsyntaxtextarea.SyntaxScheme;
 import org.fife.ui.rsyntaxtextarea.Theme;
 import org.fife.ui.rtextarea.RTextScrollPane;
+import org.reflections.Reflections;
+import org.reflections.scanners.ResourcesScanner;
 
 import javax.swing.*;
 import java.awt.*;
 import java.awt.event.InputEvent;
 import java.io.IOException;
+import java.io.InputStream;
 import java.util.Locale;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.regex.Pattern;
 
 public class RSyntaxTextAreaStyler {
 
 	private static final Logger LOG = LogManager.getLogger(RSyntaxTextAreaStyler.class);
 
 	public static void style(RSyntaxTextArea te, RTextScrollPane sp, int initialFontSize) {
 		try {
-			Theme theme = Theme
-					.load(te.getClass().getResourceAsStream((String) UIManager.get("MCreatorLAF.CODE_EDITOR_XML")));
+			AtomicReference<InputStream> input = new AtomicReference<>();

```

Comment:
I don't think you need this complex code for this. PluginLoader class extends class loader so I think you can use that to get your inputstream in a cleaner way